packages/scripts/src/create-webpack-configs.ts

```diff
@@ -193,6 +213,7 @@ export function createWebpackConfig({
         ...baseConfig,
         target,
         entry,
+        name: target,
```

Comment:
This isn't directly related to the virtual modules, but it makes the output messages from webpack clearer
https://webpack.js.org/configuration/other-options/#name